readthedocs/vcs_support/backends/bzr.py

```diff
@@ -21,7 +22,7 @@ class Backend(BaseVCS):
 
     def update(self):
         super(Backend, self).update()
-        retcode = self.run('bzr', 'status')[0]
+        retcode = self.run('bzr', 'status', warn_only=True)[0]
```

Comment:
This command is used to check the existence of the repository and call `pull` or `clone` for example.